Regression/Checksum/benchmarks_json/test_rz_laser_injection_from_lasy_file.json

```diff
@@ -1,12 +1,12 @@
 {
   "lev=0": {
-    "Br": 100277153.12954171,
-    "Bz": 2508844.115818364,
-    "Er": 3.263693682174799,
-    "Et": 3.004484377567921e+16,
-    "Ez": 0.24219246674343975,
-    "jr": 49.68134418425658,
```

Comment:
The discrepancy for `jr` here is non-negligible! Do we have an idea of the possible reason?